oceanprotocol/DAOBot
# Fix/38 project rejected

Fixes #38 .

Changes proposed in this PR:

- verify if project has bad standings i.e. `Unreported`
- update test_airtable_proposal_standings file to reflect the changes

Testing steps:
- using test_airtable_proposal_standings.test.js file:  'Validate project standings are in a good standing state' should pass
-  using airtable: make sure you have a project with at least 2 proposals of which one is in a 'Unreported' ( with the last deliverable update made more than 3 months ago) , 'Incomplete & Inactive' or 'In Dispute'
